charts/prometheus-node-exporter/templates/servicemonitor.yaml

```
@@ -13,7 +13,9 @@ spec:
   selector:
     matchLabels:
     {{- if .Values.prometheus.monitor.selectorOverride }}
-      {{ toYaml .Values.prometheus.monitor.selectorOverride | indent 6 }}
+      {{- with .Values.prometheus.monitor.selectorOverride }}
+      {{- toYaml . | nindent 6 }}
+      {{- end -}}
     {{ else }}
       {{ include "prometheus-node-exporter.selectorLabels" . | indent 6 }}
     {{- end }}
```

`with` could be used with `else`, so this just needs:

```suggestion
      {{- with .Values.prometheus.monitor.selectorOverride }}
      {{- toYaml . | nindent 6 }}
      {{- else }}
      {{ include "prometheus-node-exporter.selectorLabels" . | indent 6 }}
      {{- end }}
```